.github/workflows/pragma-linux-ci.yml

```diff
@@ -31,16 +31,41 @@ jobs:
             generators: "Unix Makefiles"
             build_dir: 'build'
     steps:
+
+      
+      - name: setup gcc-11
```

Comment:
@SlawekNowy Why is this needed?

Reply:
You can remove it. This switches gcc version to 11. Do check if runner fails with this removed.